.github/actions/test-driver/action.yml

```diff
@@ -3,6 +3,8 @@ inputs:
   junit-name:
     required: true
     default: 'driver'
+  test-args:
+    required: false
```

Comment:
Why this change?

Reply:
Because we need to provide the additional arguments to how we invoke the command and it wasn't possible to do that before.

All other steps only assume this will be run always `clojure -X:dev:ci:ee:ee-dev:drivers:drivers-dev:test`